src/chatdbg/chatdbg_ipdb.py

```diff
@@ -0,0 +1,666 @@
+"""
+in ipython_config.py:
+
+c.InteractiveShellApp.extensions = ['chatdbg.chatdbg_pdb', 'ipyflow']
+"""
+
+from colors import strip_color
+import importlib.metadata
+import atexit
+import inspect
+import os
+import pdb
+import pydoc
+import sys
+import textwrap
+import traceback 
+import json
+from datetime import datetime
+from io import StringIO
+
+import llm_utils
+
+from .assistant.assistant import Assistant
+
+from traitlets.config import Configurable
```

Comment:
This probably also needs to be added to `pyproject.toml`, I'm not familiar but doesn't seem like a built-in Python package.